package-lock.json

```diff
@@ -910,7 +910,7 @@
       }
     },
     "@babel/plugin-transform-object-assign": {
-      "version": "file:pyrene/node_modules/@babel/plugin-transform-object-assign",
+      "version": "file:https:/registry.npmjs.org/@babel/plugin-transform-object-assign/-/plugin-transform-object-assign-7.14.5.tgz",
```

Comment:
@emandilara Hello. Thanks a lot for your work ! I think the changes that you have in the` package.json` are due to the following:

1. Fist you run `npm install` in the pyrene subproject.
2. Second you run `npm install` in the root directory.

Reply:
Hi @Sakai-san, thank you for your suggestion!